packages/sdk-browser/signers/webauthn.ts

```diff
@@ -23,10 +23,9 @@ export class WebAuthnSigner implements CredentialSigner<Fido2Assertion>, Credent
     const response = await navigator.credentials.get({
       publicKey: {
         challenge: Buffer.from(challenge.challenge),
-        allowCredentials: challenge.allowCredentials.webauthn.map(({ id, type, transports }) => ({
+        allowCredentials: challenge.allowCredentials.webauthn.map(({ id, type }) => ({
           id: fromBase64Url(id),
           type,
-          transports: transports ?? [],
```

Comment:
Are these helpful at all or are browsers only using the id for lookup?

Reply:
yeah from our api, the user action challenge we return never has this set. it's always going to be `undefined`. this check here is completely useless.